.github/workflows/deploy.yml

```diff
@@ -49,9 +44,9 @@ jobs:
       elasticbeanstalk_environment: qa
       docker_tag: ${{ needs.Docker_Hub.outputs.docker_tag }}
     secrets: inherit
-
-  Production:
-    needs: [Docker_Hub, QA]
+  production:
+    name: Production
+    needs: [docker_hub, qa, qa_websocket]
```

Comment:
I don't think it'll make any practical difference but a small functional change here is that each of the three production environments depend on both `qa` and `qa_websocket`. I don't think it makes sense for `production_websocket` (for example) to depend only on `qa_websocket` and not on `qa`. `qa_websocket` and `qa` go together: the same commit is always deployed to both in parallel and both are tested together, you can't really test `qa_websocket` without also testing `qa`. If deploying to `qa_websocket` succeeded to deploying to `qa` failed it probably wouldn't be a good idea to go ahead and deploy `production_websocket`.

Really I think there should just be a single QA that runs both the web and WebSocket processes and no separate QA (WebSocket). That should be quite easy to do but it's beyond the scope of this

Reply:
I think this makes sense? If the websocket / main app is totally broken we don't want to progress on and deploy the rest. I see the same below for the websocket too.

There might be a fancier way of doing it, but honestly this seems fine to me?